From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Date: Tue, 3 May 2011 10:57:16 +0000
Subject: Re-fix bashism in ./configure: s/let/$(( ))/
Origin: http://svnview.digium.com/svn/asterisk?view=rev&rev=316193
Bug: https://issues.asterisk.org/view.php?id=17485

A forward-port in r278985 accidentally re-introduced issue 17485. Fixing
it. Thanks to Jilles Tjoelker for the good report.

Already applied in the upstream 1.8 branch. Will probably be included in
1.8.5.

---
 autoconf/ast_check_pwlib.m4 |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/autoconf/ast_check_pwlib.m4 b/autoconf/ast_check_pwlib.m4
index 98f3eb7..ecacddb 100644
--- a/autoconf/ast_check_pwlib.m4
+++ b/autoconf/ast_check_pwlib.m4
@@ -205,9 +205,9 @@ AC_DEFUN([AST_CHECK_PWLIB_VERSION], [
 		$2_VER=$((${$2_MAJOR_VERSION}*10000+${$2_MINOR_VERSION}*100+${$2_BUILD_NUMBER}))
 		$2_REQ=$(($4*10000+$5*100+$6))
 		if test "x$10" = "x"; then
-			let $2_MAX=9999999
+			$2_MAX=9999999
 		else
-			let $2_MAX=$8*10000+$9*100+$10
+			$2_MAX=$(($8*10000+$9*100+$10))
 		fi
 
 		AC_MSG_CHECKING(if $1 version ${$2_VERSION} is compatible with chan_h323)
-- 
1.7.4.4

